Content.Server/Lathe/LatheSystem.cs

```diff
@@ -63,6 +73,47 @@ public override void Update(float frameTime)
             }
         }
 
+        private void OnLatherEjectMessage(EntityUid uid, LatheComponent lathe, LatheEjectMaterialMessage message)
```

Comment:
add an "EjectCap"/"Limit" or something datafield in the LatheComponent that limits the default ejection to 1 stack, then check that against the extracted amount or something here